_posts/2014-02-15-geting-started-index.md

```diff
@@ -74,7 +74,7 @@ You'll also need to add the Sugar application to the `application/0` function in
 
 ```elixir
 def application do
-  [ applications: [ :sugar ],
+  [ applications: [],
```

Comment:
Yep, `mod` is redundant in both cases, at least on Elixir 1.4.  1.3 and earlier might be another story, though.